core/src/main/java/com/hcl/ecomm/core/services/AddToCartService.java

```diff
@@ -0,0 +1,18 @@
+package com.hcl.ecomm.core.services;
```

Comment:
@rakeshkumar-verma why would we need separate class for this ? can't we merge it into cartService ?